cmd/cogs/main.go

```diff
@@ -114,14 +127,20 @@ func run() error {
 			var modFn []func(string) string
 			// if --preserve was called, do not convert variable names to uppercase
 			if !conf.Preserve {
-				modFn = append(modFn, strings.ToUpper)
+				modFn = append(modFn, strcase.UpperSnakeCase)
```

Comment:
why this change?

Reply:
This is to handle conversions of `lowerCamelCase`, `CamelCase`, and `snake_case` to `SCREAMING_SNAKE_CASE`